README.md

```diff
@@ -14,32 +14,115 @@ of execution.
 
 * [Install](#install)
 * [API](#api)
+* [Development](#development)
 * [License](#license)
 
 
-<h2><a name="install">Install</a></h2>
+## Install
 To install mu, simply use npm,
 
+```sh
+$ npm install mu
 ```
-npm install mu
+
+## API
+
+* [Core][mu]
+* [Error Handling][mu-error]
+* Transports
+  * [mu-local][mu-local]
+  * [mu-tcp][mu-tcp]
+  * [mu-http][mu-http] (PLACEHOLDER - TODO)
+  * [mu-redis][mu-redis]
+* Adapters
+  * [mu-balancer][mu-balancer]
+  * [mu-tee][mu-tee]
+* Internals
+  * [mu-router][mu-router]
+  * [mu-transport][mu-transport]
```

Comment:
we need to put a quick and dirty example before all of this, otherwise it will be hard for people to quickly grasp it.